history-of-quakers/modernized/06-chapter-4.adoc

```diff
@@ -201,9 +219,14 @@ and I will bring deliverance another way, and you shall know that I am the Lord.
 Moved of the Lord to declare and write this, by a servant of the Truth for Jesus`'s sake,
 and a lover of your soul, called,
 
+[.signed-section-signature]
 Francis Howgill.
+
+[.signed-section-context-close]
```

Comment:
Should this NOT be marked up with SSCC since it's so long?

Reply:
context closes can be long, it's really just the the `signed-section-closing` that should be no longer than 75% of the line length. so this is fine. 👍

Reply:
ahh gotcha!